src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingUtils.kt

```diff
@@ -325,3 +329,20 @@ private fun VilkårResultat.validerVilkår_MELLOM_1_OG_2_ELLER_ADOPTERT(
         "T.o.m datoen må være lik barnets 2 års dag."
     else -> null
 }
+
+fun hentInnvilgedePerioder(
+    personopplysningGrunnlag: PersonopplysningGrunnlag,
+    vilkårsvurdering: Vilkårsvurdering
+): Pair<List<PeriodeResultat>, List<PeriodeResultat>> {
+    val periodeResultater = vilkårsvurdering.personResultater.flatMap { it.tilPeriodeResultater() }
+
+    val barnaIdenter = personopplysningGrunnlag.barna.associateBy { it.aktør }
```

Comment:
Trenger denne å være et map? 

```suggestion
    val barnaIdenter = personopplysningGrunnlag.barna.map { it.aktør }
```
Og bruke `barnaIdenter.contains(it.aktør)` på linje 351

Reply:
Og det er vel ikke identene vi har
```suggestion
    val barna = personopplysningGrunnlag.barna.associateBy { it.aktør }
```